### PR TITLE
Coach contents post bug bash fixes

### PR DESCRIPTION
**Pull request by jonboiser**
### Summary

UI fixes for coach contents

1. Show coach content icon (hereafter "Icon") on Channels for Coach Reports (recent + all activity)
1. Show Icon on Channels during exam creation
1. Show Icon on Available Channels Page (import/export)
1. Show Icon on Channels during Lesson creation
1. Fix alignment of Icon next to Questions Button in Exam Preview
1. Increase height of mobile content cards so Icon isn't squashed
1. Fix `by_role` filter so it also filters out topics that are 100% coach content @ralphiee22 

### Reviewer guidance

1. Manually go to each of the above mentioned UIs as a learner, coach, facility coach, and admin

### References

…

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by indirectlylit** [comment]
You may need to rebase to get tests to pass

**Comment by indirectlylit** [comment]
During exam creation, if a row is disabled, the coach content icon should get the 'disabled' treatment also:

image

**Comment by indirectlylit** [comment]
would it be challenging to make the tooltip more explicit? E.g. "Includes 3 coach resources"

image

File: kolibri/plugins/learn/assets/src/views/content-card/card.styl [hunk 1]
```diff
@@ -5,4 +5,4 @@ $thumb-width-desktop = 210px
 $thumb-height-desktop = round($thumb-width-desktop / $ratio)
 
 $thumb-width-mobile = 150px
-$thumb-height-mobile = round($thumb-width-mobile / $ratio)
+$thumb-height-mobile = max(round($thumb-width-mobile / $ratio), 92px)
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/views/content-card/card.styl:8]: This is going to make the thumbnails a little bit off the target `$ratio`

If the goal is to set a particular height of `92px`, you could do something like

```stylus
$thumb-height-mobile = 92px
$thumb-width-mobile = round($thumb-height-mobile * $ratio)
```

**Comment by rtibbles** [comment]
@jonboiser could you rebase/merge this to let the Python tests run?

**Pull request merged**